allennlp/models/model.py

```diff
@@ -348,20 +348,20 @@ def load(
 
         # Parameters
 
-        config: Params
+        config : `Params`
             The configuration that was used to train the model. It should definitely
             have a `model` section, and should probably have a `trainer` section
             as well.
-        serialization_dir: str = None
+        serialization_dir: `str = None`
```

Comment:
Do you need a space before the colon? `:` I'm not sure.

Reply:
Either way works